src/Auth.php

```diff
@@ -11,6 +11,8 @@ class Auth {
 	protected static $issued;
 	protected static $expiration;
 	protected static $is_refresh_token = false;
+	protected static $refresh_token_valid_days = 30;
```

Comment:
I think 30 days is probably a fine default instead of a year. If you're not using an application for more than 30 days, the application asking you to login again seems reasonable.